getting_started/build-guide/build-from-scratch.rst

```diff
@@ -155,43 +160,45 @@ Build and install the headers for each platform.
 .. code-block:: bash
 
    #Baytrail
-   $ ./configure --target=xtensa-byt-elf --prefix=~/work/sof/xtensa-root
+   $ ./configure --target=xtensa-byt-elf --prefix=/home/$USER/work/sof/xtensa-root
    $ make
    $ make install
    #Haswell
-   $ ./configure --target=xtensa-hsw-elf --prefix=~/work/sof/xtensa-root
+   $ ./configure --target=xtensa-hsw-elf --prefix=/home/$USER/work/sof/xtensa-root
    $ make
    $ make install
    #Apollo Lake
-   $ ./configure --target=xtensa-apl-elf --prefix=~/work/sof/xtensa-root
+   $ ./configure --target=xtensa-apl-elf --prefix=/home/$USER/work/sof/xtensa-root
```

Comment:
What's the reason for changing ~/ here ?

Reply:
configure script expects full path not relative path. Maybe that requirement was changed after the document.